src/Gremlin.Net.CosmosDb/GraphTraversalSource.cs

```diff
@@ -7,7 +7,7 @@ namespace Gremlin.Net.CosmosDb
     /// A graph traversal source (g)
     /// </summary>
     /// <seealso cref="Gremlin.Net.CosmosDb.IGraphTraversalSource"/>
-    internal sealed class GraphTraversalSource : IGraphTraversalSource
+    public sealed class GraphTraversalSource : IGraphTraversalSource
```

Comment:
Any reason why you needed to make this public? Can this stay internal? I was trying to keep some of the implementation details hidden from client applications using the code.

Reply:
It only was because I needed to remove the line of AssemblyInfo.cs that involved that Tests project could see the internals of the other project, and then there was an error because GraphTraversalSource is referenced somewhere outside its dll.
Only for this, I do not need this change explicitly for anything but compilling the project, if you solve this in another way, it is perfect for me.
Thanks in advance.